src/components/Home/container.js

```diff
@@ -97,7 +98,10 @@ export default Container => class extends Component {
     }
 
     formatDate = data => data.reduce((obj, item) => {
-      const holidayStatus = holidayStatusColor[item.eventStatus.eventStatusId];
+      const isWFH = (item.eventType.description === eventType.WFH);
```

Comment:
in keeping this consistent can we go with `isWfh` instead of `isWFH` also no brackets needed here :)

Reply:
done :)